src/dashboard/src/main/management/commands/rebuild_elasticsearch_aip_index_from_files.py

```diff
@@ -0,0 +1,287 @@
+"""Recreate the Elasticsearch index from AIPs stored on disk.
+
+This is a copy of: https://git.io/vxadg.
```

Comment:
The shortened url here doesn't work for me.  I suggest putting in the full url, rather than use a shortener.  I am guessing you are trying to link to something like https://github.com/artefactual/archivematica-devtools/blob/master/tools/rebuild-elasticsearch-aip-index-from-files

Reply:
Fixed.